cocotb/share/lib/simulator/simulatormodule.cpp

```diff
@@ -926,133 +926,152 @@ static PyMethodDef gpi_sim_hdl_methods[] = {
         (PyCFunction)get_signal_val_long, METH_NOARGS, PyDoc_STR(
             "get_signal_val_long($self)\n"
             "--\n\n"
-            "Get the value of a signal as a long"
+            "get_signal_val_long(self) -> int\n"
```

Comment:
Somewhere in this file we should have a comment explaining why the signature appears twice - the first is to populate `__text_signature__` which is part of python itself, while the latter is simply the first line of the docstring which is parsed by `sphinx.ext.autodoc`.